example_data/config_bark.yaml

```diff
@@ -0,0 +1,31 @@
+input_file: "example_data/a.md"
+output_folder: "example_data/bark/"
+text_to_text_model: "allenai/OLMoE-1B-7B-0924-Instruct-GGUF/olmoe-1b-7b-0924-instruct-q8_0.gguf"
+text_to_speech_model: "suno/bark"
+text_to_text_prompt: |
+  You are a podcast scriptwriter generating engaging and natural-sounding conversations in JSON format. 
+  The script features the following speakers:
+  {SPEAKERS}
+  Instructions:
+  - Write dynamic, easy-to-follow dialogue.
+  - Include natural interruptions and interjections.
+  - Avoid repetitive phrasing between speakers.
+  - Format output as a JSON conversation.
+  Example:
+  {
+    "Speaker 1": "Welcome to our podcast! Today, we're exploring...",
+    "Speaker 2": "Hi! I'm excited to hear about this. Can you explain...",
+    "Speaker 1": "Sure! Imagine it like this...",
+    "Speaker 2": "Oh, that's cool! But how does..."
+  }
+speakers:
+  - id: 1
+    name: Laura
+    description: The main host. She explains topics clearly using anecdotes and analogies, teaching in an engaging and captivating way.
+    voice_profile: "v2/en_speaker_0"
+
+  - id: 2
+    name: Daniel
+    description: The co-host. He keeps the conversation on track, asks curious follow-up questions, and reacts with excitement or confusion, often using interjections like hmm or umm.
+    voice_profile: "v2/en_speaker_1"
+outetts_language: "en"  # Supported languages in version 0.2-500M: en, zh, ja, ko.
```

Comment:
I think it is ok to not include the value in the config because there is a default value in the schema. So, in cases like this where outetts is not used, there is no need to include it

Reply:
makes sense, i ll remove it

Reply:
wasn't removed